src/actions/getCurrentVersionByPackage.js

```diff
@@ -9,7 +9,7 @@ export function getCurrentVersionByPackage({ config, npm, props }) {
       .map(name =>
         npm.getCurrentPackageVersion(name).then(version => ({
           name,
-          version,
+          version: version || '1.0.0',
```

Comment:
I think we should leave it as `null` here. We need the `evaluateNewVersion` to bump to `1.0.0`.

Otherwise, if we have changes, the initial release version will be computed as `1.0.1` or `1.1.0` or even `2.0.0`.